common/elasticsearch/src/main/scala/uk/ac/wellcome/elasticsearch/ImagesIndexConfig.scala

```diff
@@ -7,18 +7,23 @@ import com.sksamuel.elastic4s.requests.mappings.dynamictemplate.DynamicMapping
 
 object ImagesIndexConfig extends IndexConfig with WorksIndexConfigFields {
```

Comment:
Do we need separate index config per state :thinking:? Suppose not required until we move images part of the pipeline to pipeline storage

Reply:
Yeah that was my logic